examples/account/evmapp/src/main/java/io/horizen/examples/EvmApp.java

```diff
@@ -12,6 +12,9 @@ public class EvmApp {
     public static void main(String[] args) {
         if (args.length == 0) {
             System.out.println("Please provide settings file name as first parameter!");
+            System.out.println("Optional parameters:");
+            System.out.println("MC Block Reference delay (int)");
+            System.out.println("All forks enabled (boolean, default false");
```

Comment:
let's be a bit more clear:
All forks enabled from earliest epoch (2)
(also in the other config all the forks are enabled , but on different epochs..)

Reply:
Done